src/Model/Payment/Shopgate/Shopgate.php

```diff
@@ -30,9 +30,10 @@
 
 class Shopgate extends AbstractPayment
 {
-    const MODULE_NAME            = 'Shopgate_Base';
-    const PAYMENT_CODE           = 'shopgate';
-    const XML_CONFIG_STATUS_PAID = 'payment/shopgate/order_status';
+    const MODULE_NAME                = 'Shopgate_Base';
+    const PAYMENT_CODE               = 'shopgate';
+    const XML_CONFIG_STATUS_PAID     = 'payment/shopgate/order_status';
+    const XML_CONFIG_STATUS_NOT_PAID = 'payment/shopgate/order_status';
```

Comment:
what happened to destructive coding? :( 
Also, if you leave one, may just need to change to ORDER_STATUS